### ISSUE TITLE
Diagnostics are not displayed for invalid values, if there is an empty line before the variable.

### ISSUE THREAD

**Issue by aparnamichael** [OpenLiberty/liberty-tools-intellij#1017]
Steps to reproduce:

1. Add a variable "WLP_LOGGING_CONSOLE_FORMAT=DEV".
2. Add an empty line (\n)
3. Add another variable and assign an invalid value. "WLP_LOGGING_CONSOLE_LOGLEVEL=NONE"
4. The diagnostic is not showing for the invalid value - "NONE"

Tracking https://github.com/OpenLiberty/liberty-language-server/pull/310

LSP Traces:

1. If there is an extra '\n' in "text" then  'textDocument/publishDiagnostics' is not showing in the Traces.
```
[Trace - 11:52:58] Sending notification 'textDocument/didChange'
Params: {
  "textDocument": {
    "version": 2,
    "uri": "file:///Users/aparnacmichael/Documents/test-projss/app-name-test-one/src/main/liberty/config/server.env"
  },
  "contentChanges": [
    {
      "text": "WLP_LOGGING_CONSOLE_FORMAT\u003dDEV\n\nWLP_LOGGING_CONSOLE_LOGLEVEL\u003dNONE"
    }
  ]
}
```

2. In this scenario there is no '\n' or empty line in between the variables and 'textDocument/publishDiagnostics' is present in the traces.
```
[Trace - 11:48:41] Sending notification 'textDocument/didChange'
Params: {
  "textDocument": {
    "version": 2,
    "uri": "file:///Users/aparnacmichael/Documents/test-projss/app-name-test-one/src/main/liberty/config/server.env"
  },
  "contentChanges": [
    {
      "text": "WLP_LOGGING_CONSOLE_FORMAT\u003dDEV\nWLP_LOGGING_CONSOLE_LOGLEVEL\u003dNONE"
    }
  ]
}


[Trace - 11:48:41] Received notification 'textDocument/publishDiagnostics'
Params: {
  "uri": "file:///Users/aparnacmichael/Documents/test-projss/app-name-test-one/src/main/liberty/config/server.env",
  "diagnostics": [
    {
      "range": {
        "start": {
          "line": 1,
          "character": 29
        },
        "end": {
          "line": 1,
          "character": 33
        }
      },
      "severity": 1,
      "source": "Liberty Config Language Server",
      "message": "The value `NONE` is not valid for the variable `WLP_LOGGING_CONSOLE_LOGLEVEL`."
    }
  ]
}

```
https://github.com/user-attachments/assets/88785124-676b-44d3-a3fe-a725108acf04

vs code behaviour is a little different but similar issue is present in vs code as well -https://github.com/OpenLiberty/liberty-tools-vscode/issues/343

**Comment by angelozerr** [comment]
It is a problem with your language server and not from LSP4IJ. You should have the same problem with vscode.

**Comment by arunvenmany-ibm** [comment]
Hi @cherylking 

I found the issue is due to blank line not having any property and causing NPE.
Is it okay to ignore the blank line and just scan the next line ?
Is there any requirement to show diagnostics on blank line? 
I made some code changes for skipping blank line and its working fine now


![image](https://github.com/user-attachments/assets/11784160-6f39-4cf2-8148-653c0ed61c63)

**Comment by cherylking** [comment]
@arunvenmany-ibm According to this [documentation](https://openliberty.io/docs/latest/reference/config/server-configuration-overview.html#server-env), empty lines should be ignored.

**Comment by dessina-devasia** [comment]
The issue has been resolved with LCLS 2.2-SNAPSHOT in LTI. Attaching screenshots for the reference.

![image](https://github.com/user-attachments/assets/51564d4a-0a1c-425f-a66e-f47cd195fd86)

![image](https://github.com/user-attachments/assets/0c9e56dd-7410-4296-a29d-aee66a8df223)
